tests/test_builders/test_build_warnings.py

```diff
@@ -92,7 +113,7 @@ def test_html_warnings_exception_on_warning(app: SphinxTestApp) -> None:
         pytest.fail('Expected an exception to be raised')
     except SphinxError:
         tb = traceback.format_exc()
-        assert 'unindent_warning' in tb
```

Comment:
I've ~~dropped~~ removed this assertion by mistake; I'll restore that in a few moments.

Edit: rephrase to reduce ambiguity